flow/FileTraceLogWriter.cpp

```diff
@@ -79,18 +83,14 @@ void FileTraceLogWriter::write(const std::string& str) {
 			remaining -= ret;
 			ptr += ret;
 		} else {
-			issues.insert(LiteralStringRef("trace_log_writer_flush_failure"));
+			issues->addIssue("trace_log_writer_flush_failure");
```

Comment:
I think this could add two issues to the set because of the call to `lastError` below. 

You could arguably call it something like `trace_log_file_write_failure`, since we technically aren't flushing here, though I'm not particularly opposed to `flush`. Changing `writer` to `file` I think is a little more clear because `writer` is an internal class name.